internal/tui/sync_server.go

```
@@ -0,0 +1,159 @@
+
+	return nil, fmt.Errorf("no available ports in range %d-%d", startPort, startPort+maxAttempts-1)
+}
+
+func (s *SyncServer) GetPort() int {
+	return s.port
+}
+
+func (s *SyncServer) Start() {
+	s.running = true
+
+	go s.acceptConnections()
```

golangci-lint (1.64.8)

Error return value of `s.acceptConnections` is not checked

(errcheck)